src/otx/algo/segmentation/heads/base_segm_head.py

```diff
@@ -16,7 +16,7 @@
 from otx.algo.utils.mmengine_utils import load_checkpoint_to_model, load_from_http
 
 
-class BaseSegmHead(nn.Module, metaclass=ABCMeta):
+class BaseSegmHead(nn.Module):
```

Comment:
How about not using abbreviations? `BaseSegmHead` -> `BaseSegmentationHead`

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3863